src/app/wallet/wallet.module.ts

```diff
@@ -25,7 +26,8 @@ import { routing } from './wallet.routing';
     OverviewComponent,
     SettingsComponent,
     StakinginfoComponent,
-    ColdstakeComponent
+    ColdstakeComponent,
+    ConsoleModalComponent
```

Comment:
Just make the console, it's own module from Core-UI. You should be able to just use it, without having to import it I believe.